### ISSUE TITLE
Expected Two LoRAs (High-Noise/Low-Noise DIT) for Wan2.2, But Only One Generated in Output Directory

### ISSUE THREAD

**Issue by JamesTensor** [kohya-ss/musubi-tuner#552]
Thank you for your world-class work! I have been using this project consistently. I intended to train two LoRA models (one for high-noise DIT and one for low-noise DIT) for Wan2.2, but in reality, I only found one LoRA model file in the output_dir. Moreover, I don’t know whether this file corresponds to the high-noise DIT or the low-noise DIT. Could you tell me if I should expect to get two LoRA models?

Below is my model training script:
nohup accelerate launch --num_cpu_threads_per_process 1 --mixed_precision fp16 src/musubi_tuner/wan_train_network.py --task t2v-A14B --dit /root/pretrained_model/wan2.2_t2v_low_noise_14B_fp16.safetensors/split_files/diffusion_models/wan2.2_t2v_low_noise_14B_fp16.safetensors --dit_high_noise /root/pretrained_model/wan2.2_t2v_high_noise_14B_fp16.safetensors/split_files/diffusion_models/wan2.2_t2v_high_noise_14B_fp16.safetensors --dataset_config /root/src/musubi_tuner/dataset/moshou/lora_config.toml --mixed_precision fp16 --optimizer_type AdamW8bit --learning_rate 2e-4 --gradient_checkpointing --max_data_loader_n_workers 1 --persistent_data_loader_workers --network_module networks.lora_wan --network_dim 32 --timestep_sampling shift --discrete_flow_shift 12.0 --min_timestep 0 --max_train_epochs 1000 --timestep_boundary 875 --seed 42 --preserve_distribution_shape --output_dir /root/lora_train/moshou_0621_t2v --output_name moshou --fp8_scaled --fp8_base --vae_cache_cpu --save_every_n_steps 300 --flash_attn --split_attn --gradient_accumulation_steps 4 > /root/log/wan_moshou_t2v.txt 2>&1 &

Thank you for your help!

**Comment by m-i-i-m** [comment]
I think your training script is set up for dual training, that's why you are only getting one lora, that is to be used for both low and high models when inferencing.
From the docs>
"If you want to train LoRA for both models simultaneously, you need to specify the low-noise model with --dit and the high-noise model with --dit_high_noise. The two models are switched at the timestep specified by --timestep_boundary. The default value is 0.9 for I2V and 0.875 for T2V. --timestep_boundary can be specified in the range of 0.0 to 1.0, or in the range of 0 to 1000.

When training Wan2.2 high and low models, you can use --offload_inactive_dit to offload the inactive DiT model to the CPU, which can save VRAM (only works when --blocks_to_swap is not specified)."

**Comment by JamesTensor** [comment]
Thank you for your reply!! As you mentioned, I trained both the high-noise and low-noise DiTs simultaneously and obtained only one LoRA model. Then, during inference, can this single LoRA model be applied to both the high-noise DiT and the low-noise DiT? And why is it possible for one LoRA to work on two DiTs with different parameters?

**Comment by m-i-i-m** [comment]
I'm not an expert in this matter. On the surface,  I'm guessing since it is trained together it is writing data to the same file. Where as the other method you train each model separately and not specifying the "--dit_high_noise" parameter.
